autoscheduler/scraper/management/commands/utils/scraper_utils.py

```diff
@@ -2,51 +2,22 @@
 from typing import List
 from itertools import product
 
-def get_all_terms(year: int = -1, now=datetime.now()) -> List[str]:
+def get_all_terms(*_, year: int = None, now=datetime.now()) -> List[str]:

```

Comment:
![image](https://user-images.githubusercontent.com/7295783/120935035-c0259200-c6b5-11eb-9c38-33622122c5a1.png)

mfw when you reduce my 40 line function to 8 lines

Reply:
To be fair it's because I made it so the function doesn't need to handle edge cases anymore since terms without courses are ignored, but it's always nice when something ends up being simplified in a PR